### ISSUE TITLE
Blocked aria-hidden on a <a> element with the Grid widget

### ISSUE THREAD

**Issue by eyupyusein** [telerik/kendo-ui-core#7960]
### Bug report

There is a javascript error with the aria-hidden attribute in some Grid scenarios. Probably related to:
https://source.chromium.org/chromium/chromium/src/+/62d86ddddc8fad157321e42e2acdb18b6ebba951

### Reproduction of the problem

Reproducible in the live demos as well (on Chromium browsers):

- Filter menu icon:
https://demos.telerik.com/kendo-ui/grid/filter-menu-customization
- Hierarchy expand/collapse button:
https://demos.telerik.com/kendo-ui/grid/hierarchy  

Steps:
1. Open this demo in latest version of Chrome and Incognito window:
https://demos.telerik.com/kendo-ui/grid/hierarchy

2. Open the console and click on Collapse button.

3. You will notice the following javascript error:

![image](https://github.com/user-attachments/assets/b84b5c41-e8d9-45c7-9781-8699acc70596)

Full error:
_Blocked aria-hidden on a <a> element because the element that just received focus must not be hidden from assistive technology users. Avoid using aria-hidden on a focused element or its ancestor. Consider using the inert attribute instead, which will also prevent focus. For more details, see the aria-hidden section of the WAI-ARIA specification at https://w3c.github.io/aria/#aria-hidden. <a href=​"#" aria-label=​"Collapse" tabindex=​"-1" class=​"k-icon k-svg-icon k-svg-i-caret-alt-down" aria-hidden=​"true">​</a>_

**Expected Result:** The _aria-hidden_ attribute should be removed

**Comment by mihaela-lukanova** [comment]
Reported in TickedId: 1665179

**Comment by danvitoriano** [comment]
hello, I did not understand: is this problem solved? cause it still happening to me
